core/arch/aarch64/codec.c

```diff
@@ -940,6 +940,33 @@ encode_opnd_dq5(uint enc, int opcode, byte *pc, opnd_t opnd, OUT uint *enc_out)
     return encode_opnd_dq_plus(0, 5, 30, opnd, enc_out);
 }
 
+/*
+ * dq16: D/Q register at bit position 16 with 4 bits only, for the FP16
```

Comment:
This should be "dq16_fsz16"?

Reply:
Yep, done. I am not entirely happy with the name fsz16 overall, I think we could have a better name that makes it more clear that it expresses the element width for half precision. But that's for another change.